benchmark/src/generation.rs

```diff
@@ -218,6 +218,6 @@ fn create_sequence(sequence_length: u32, tokenizer: Tokenizer) -> String {
     encoding.truncate(sequence_length as usize, 0, TruncationDirection::Left);
     // Decode
     tokenizer
-        .decode(Vec::from(encoding.get_ids()), false)
+        .decode(encoding.get_ids().as_ref(), false)
```

Comment:
```suggestion
        .decode(From::from(encoding.get_ids()), false)
```

You see the error because the version of `tokenizers` installed from `Dockerfile` is different from the one in `Cargo.lock`. We want the implementation be compatible with both versions until we update `Cargo.lock`.